.gitignore

```diff
@@ -250,3 +250,8 @@ paket-files/
 # JetBrains Rider
 .idea/
 *.sln.iml
+Build
```

Comment:
There is a section in the gitignore with excldue for build x32 and x64. Please enhance the section there
https://github.com/Yuri05/OSPSuite.FuncParser/blob/a34f73a4314a4ce5046b187bdf963ef363a07ac8/.gitignore#L13